airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionNotificationWorkflowImpl.java

```diff
@@ -35,14 +34,14 @@ public class ConnectionNotificationWorkflowImpl implements ConnectionNotificatio
   @Override
   public boolean sendSchemaChangeNotification(final UUID connectionId)
       throws IOException, InterruptedException, ApiException, ConfigNotFoundException, JsonValidationException {
-    final StandardSync standardSync = configFetchActivity.getStandardSync(connectionId);
+    final Optional<Boolean> breakingChange = configFetchActivity.getBreakingChange(connectionId);
     final Optional<SlackNotificationConfiguration> slackConfig = slackConfigActivity.fetchSlackConfiguration(connectionId);
-    if (slackConfig.isPresent()) {
+    if (slackConfig.isPresent() && breakingChange.isPresent()) {
```

Comment:
Do we need to check the NonBreakingChange preference?

Reply:
no, at this point we're just using the breakingChange boolean